doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -478,6 +495,17 @@ To use the Simple GPIO implementation of FEM with SKY66112-11, complete the foll
 
    The state of the other control pins should be set according to the SKY66112-11 documentation.
    See the official `SKY66112-11 page`_ for more information.
+#. On nRF53 devices, you must also apply the same devicetree node from Step 1 to the network core.
+   To do so, apply the overlay to the correct network core child image by creating an overlay file named :file:`child_image/*childImageName*.overlay` in your application directory, for example :file:`child_image/multiprotocol_rpmsg.overlay`.
```

Comment:
```suggestion
#. On nRF53 devices, you must also apply the same devicetree node to the network core.

   Create a devicetree overlay file with the same information as you used in Steps 1 to 3.
   To apply the overlay to the correct network core child image, create the file in the :file:`child image` directory of your application directory, and name it :file:`*childImageName*.overlay`, for example :file:`child_image/multiprotocol_rpmsg.overlay`.
```
I believe this is giving the correct information in a bit more structured manner. Could also remove the empty line after this, so the line introducing the possible names is part of the same paragraph.

Reply:
Done.